TACC/Core-Styles
# feat: offer bootstrap form fixes to portal client

## Overview

1. Add `bootstrap.form.css` into `core-styles.portal.css`.
2. Demo Bootstrap `input-group-prepend` plus TACC/Core-Styles `c-button`.
3. Fix Bootstrap `input-group-prepend` plus TACC/Core-Styles `c-button`.

## Related

- fixes issue B1 of https://github.com/TACC/tup-ui/pull/246#issuecomment-1623976572

## Changes

- feat: demo bootstrap form with core styles button
- fix: bootstrap input prepend + core styles button
- feat: load bootstrap form css for portal clients

## Testing

0. `npm start`
1. Open http://localhost:3001/components/detail/bootstrap--form.
2. Scroll to "Bootstrap + Core Styles" "Best Friend".
3. Verify button height and input height match.
4. Verify button and input share one border.
5. Verify button dark border is above input light border.

## UI

<img width="1240" alt="bootstrap form and core styles" src="https://github.com/TACC/Core-Styles/assets/62723358/a5a6c978-4342-4722-8578-64838077d195">
